src/main/java/library/typeOfOnlineLibrary/people/customer/Customer.java

```diff
@@ -0,0 +1,24 @@
+package library.typeOfOnlineLibrary.people.customer;
+
+import library.typeOfOnlineLibrary.people.stuff.People;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public  class Customer extends People {
```

Comment:
perhaps it is better extends from Human instead of People?

Reply:
FIXED